src/ui/actions/timeline.ts

```diff
@@ -125,7 +126,9 @@ export function jumpToInitialPausePoint(): UIThunkAction {
       hasFrames = initialPausePoint.hasFrames;
       time = initialPausePoint.time;
     }
-    ThreadFront.timeWarp(point, time, hasFrames);
+    if (isPointInLoadingRegion(state, point)) {
```

Comment:
what's this if statement doing?

Reply:
If we are not actually loading the point that this would take us to, we should not try and time warp there, it will just cause a protocol error. If the URL also contained a focus window, then the point will be inside the range and we'll be all good 👍

Reply:
This is a little unrelated to the focus window stuff, but I spotted it and figured while I was in here I would be a little more defensive about pause creation.